rollup.config.js

```diff
@@ -0,0 +1,50 @@
+import { join } from 'path';
+import babel from 'rollup-plugin-babel';
+import uglify from 'rollup-plugin-uglify';
+
+import pkg from './package.json';
+
+const isProd = process.env.NODE_ENV === 'production';
+
+export default {
+  input: join(__dirname, 'src/index.js'),
+  output: [
+    {
+      file: isProd ? './lib/redux-observable.cjs.min.js' : './lib/redux-observable.cjs.js',
+      format: 'cjs',
+      sourcemap: true,
+    },
+    {
+      file: isProd ? './lib/redux-observable.esm.min.js' : './lib/redux-observable.esm.js',
+      format: 'es',
+      sourcemap: true,
+    },
+    {
+      file: isProd ? './dist/redux-observable.umd.min.js' : './dist/redux-observable.umd.js',
+      format: 'umd',
+      name: 'ReduxObservable',
+      sourcemap: true,
```

Comment:
You can add globals here to suppress the warning `(!) Missing global variable names`